src/PipelineNet.ServiceProvider.Tests/MiddlewareResolver/ActivatorUtilitiesMiddlewareResolverTests.cs

```diff
@@ -200,6 +200,22 @@ public void Resolve_TransientServiceGetsDisposed()
             Assert.True(middleware.Service.Disposed);
         }
 
+        [Fact]
+        public void Resolve_TransientServiceGetsDisposed2()
```

Comment:
Maybe something like this could help to remove the `1` and `2` suffix in the methods:

```suggestion
        public void Resolve_TransientServiceGetsDisposedWhenScopeIsDisposed()
```

Reply:
Sure, could be:
- Resolve_TransientServiceGetsDisposedWhenScopeIsDisposed
- Resolve_TransientServiceGetsDisposedWhenRootServiceProviderIsDisposed

The PR is closed now though.